charts/teuto-cnpg/templates/podmonitor.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    cnpg.io/cluster: {{ printf "%s" .Release.Name }}
+    managed.by/monitoring: teutonet
```

Comment:
This should be configurable? like it is in https://github.com/teutonet/teutonet-helm-charts/blob/main/charts/base-cluster/values.yaml#L219-L222

That way the user is not forced to use our base-cluster / this specific label